### DISCUSSION TITLE
huge fonts on dropdowns in menu

### DISCUSSION THREAD

**Discussion by EdoFro** [freeplane/freeplane#276] [Help] [answered]
Originaly posted in [SF Forum](https://sourceforge.net/p/freeplane/discussion/758437/thread/26105bd504/?limit=25#bffd):

> Hello,
I like Freeplane very much and use it a lot, so thanks to the programmers.
I have an issue with the font size in dropdowns in menue bar, the same font size appears in editor dropdowns and also in the whole preferences window the about window and others. The font size of standard menu entries is ok and I can control it via preferences window. Also I have the feeling the mentioned font sizes got larger recently so it seems Freeplane takes this font (size) from some other windows settings (just a guess).
I hope you know what I mean I attach a scrren shot to show you what I mean
Thanks for any help
Martin

![image](https://user-images.githubusercontent.com/35700575/160221625-0b9aa178-4b36-4a1e-bf44-a33fd129448d.png)

**Comment by EdoFro** [comment, marked as answer]
Hi,
the UI Font size depends on the monitor size too.

look at the following preferences image and test changing the three marked preferences:

![fontSizeInMenus](https://user-images.githubusercontent.com/35700575/160221816-017c5b19-81cb-4266-b786-ea29356e2f50.png)


For example, the fonts gets smaller when enlarging the value of the monitor size.

hope it helps

**Comment by MrMfromK** [comment]
Hi Edo,
thanks for your answer.
I played with these too but nothing helped. But when I restarted my computer after the weekend the problem was gone, Looks normal (again) now. So I guess it was some windows or java setting gone wild.
